src/ECAA80.c

```diff
@@ -25,7 +25,16 @@ void ECAA80_UnkFloAI_Chase(Evt* script, MobileAISettings* aiSettings, EnemyDetec
 INCLUDE_ASM(s32, "ECAA80", func_8024199C_ECB69C);
 
 // ECAA80_ItemChoice_SaveSelected (needs data?)
```

Comment:
Can you rename `func_802419F0_ECB6F0` to `ECAA80_ItemChoice_SaveSelected`? Should be a find+replace